src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
line no: 3 - There should be a space between method name and opening parenthesis.
line no: 5 - Consider using .equals() method to compare string contents instead of == operator.